ui/src/components/user.tsx

```diff
@@ -412,6 +413,15 @@ export class User extends Component<any, UserState> {
                 )}
               </ul>
             </h5>
+            <div className="d-flex align-items-center mb-2">
```

Comment:
I'll test this out, I'm guessing its good.

Reply:
Yeah I think this should be fine, as it's just vertically centering the cake icon and cake day label in user overview page.